<launch>
  <!-- The name of the object we record -->
  <arg name="model_name"/>

  <!-- The PCL descriptor used for training:
     1.- PFHRGB
     2.- SHOTRGB-->
  <arg name="descriptor_type" default="1"/>

  <!-- The operation to perform 
     1.- Record raw data
     2.- Compute models from raw data-->
  <arg name="operation"/>

  <!-- RGBD camera -->
  <include file="$(find openni2_launch)/launch/openni2.launch">
	<arg name="depth_registration" value="true" />
</include>

  <!-- Training node -->  
  <param name="model_name" value="$(model_name)"/>
  <param name="descriptor_type" value="$(descriptor_type)"/>
  <param name="operation" value="$(operation)"/>

  <node pkg="object_recognition" type="vision_training" name="vision_tracking" output="screen">	
  </node>
</launch>
